src/diffusers/models/unet_2d_condition.py

```diff
@@ -279,7 +278,8 @@ def forward(
         # 6. post-process
         # make sure hidden states is in float32
         # when running in half-precision
-        sample = self.conv_norm_out(sample.float()).type(sample.dtype)
```

Comment:
need to verify that this gives identical results

Reply:
Please check https://github.com/huggingface/diffusers/pull/371#issuecomment-1259478169